src/query.rs

```diff
@@ -53,6 +63,9 @@ impl Query {
     /// [`Identifier`]: crate::sql::Identifier
     #[track_caller]
     pub fn bind(mut self, value: impl Bind) -> Self {
+        if self.raw {
```

Comment:
In general, I'm fine with this implementation.

However, it's slightly strange to check in runtime properties that can easily be checked at compile time.

Reply:
Oh, it's probably duplicate of https://github.com/ClickHouse/clickhouse-rs/pull/243#discussion_r2157225738